docs/solidity-by-example.rst

```diff
@@ -89,11 +89,10 @@ of votes.
         function giveRightToVote(address voter) public {
             // If the argument of `require` evaluates to `false`,
             // it terminates and reverts all changes to
-            // the state and to Ether balances. It is often
-            // a good idea to use this if functions are
-            // called incorrectly. But watch out, this
-            // will currently also consume all provided gas
-            // (this is planned to change in the future).
+            // the state and to Ether balances. 
+            // It will also refund the remaining gas to the caller.
```

Comment:
Not sure if we should mention this explicitly, it will not refund gas right away, it will just not consume all gas (which is actually not a special case).

Reply:
So, it's ok if I change the comment to just say require wont consume gas?

Reply:
Yes!